.github/workflows/turbo-deploy.yml

```
@@ -0,0 +1,229 @@
+    runs-on: ubuntu-latest
+    needs: detect-config
+    if: needs.detect-config.outputs.deploy-type == 'auto' || (needs.detect-config.outputs.deploy-type == 'manual' && contains(fromJson('["dev", "pre", "live"]'), needs.detect-config.outputs.environment))
+    environment: ${{ needs.detect-config.outputs.environment }}
+    strategy:
+      matrix:
+        project: ${{ fromJson(needs.detect-config.outputs.projects) }}
+      fail-fast: false
+    defaults:
+      run:
+        working-directory: ./client/products/${{ matrix.project }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)